src/UI/templates/default/Item/item.less

```diff
@@ -0,0 +1,140 @@
+.il-item {
+	background-color: @il-primary-container-bg;
+	position: relative;
+	padding: @il-item-padding;
+
+	h5 {
+		margin: @il-item-h5-margin;
+		font-size: @font-size-h5;
+		float: left;
+	}
+
+	.il-item-divider {
+		clear: both;
+		padding: @il-item-divider-padding;
+		margin: @il-item-divider-margin;
+		border: 0 none;
+		background: none;
+		border-bottom: @il-item-divider-border-bottom;
+	}
+
+	.dropdown {
+		float: right;
+	}
+
+	.il-item-description {
+		padding: @il-item-description-padding;
+		font-size: @font-size-small;
+		clear: both;
+	}
+
+	.il-item-property-name {
+		font-size: @font-size-small;
+	}
+
+	.il-item-property-value {
+		font-size: @font-size-small;
+		color: @il-light-text-color;
+	}
+}
+
+.il-item-marker {
+	border-left: @il-item-marker-border-left;
+}
+
+.il-narrow-content .il-item:hover {
+	background-color: @il-highlight-bg;
+}
+
+.il-item-group h4 {
+	float:left;
+}
+
+.il-item-group .dropdown {
+	float:right;
+}
+
+.il-item-group-items {
+	clear: both;
+}
+
+/* this is currently experimental, could be removed */
```

Comment:
Yes, I think the discussion is currently open how to deal with different levels of context awareness. I think we would need a more general guideline if we agree on possible solutions.